### ISSUE TITLE
Duplicate records in observationunits endpoint 

### ISSUE THREAD

**Issue by dlebauer** [terraref/brapi#40]
```sh
curl --insecure -o observations_plot1311.json 'https://brapi.workbench.terraref.org/brapi/v1/observationunits?observationVariableDbId=6000000196&observationUnitDbId=MAC%20Field%20Scanner%20Season%204%20Range%2013%20Column%2011'
```

Appears to return duplicate records (same value, same observationDbId):  

```json
       "observations": [
          {
            "observationDbId": "6001965924",
            "observationTimeStamp": "2017-09-15T05:00:00Z",
            "observationVariableDbId": "6000000196",
            "observationVariableName": "aboveground_dry_biomass",
            "operator": "Newcomb, Maria",
            "quality": "unchecked",
            "replicate": 6000031146,
            "value": "25900"
          },
          {
            "observationDbId": "6001965924",
            "observationTimeStamp": "2017-09-15T05:00:00Z",
            "observationVariableDbId": "6000000196",
            "observationVariableName": "aboveground_dry_biomass",
            "operator": "Newcomb, Maria",
            "quality": "unchecked",
            "replicate": 6000031146,
            "value": "25900"
          }
```

Need to correct this

**Comment by dlebauer** [comment]
Seems that the original example isn't working

But this endpoint also isn't filtering by parameters observationDbId, observationunitDbId, observationunitName 
- http://localhost:5000/brapi/v1/observationunits?observationDbId=6001965924

**Comment by dlebauer** [comment]
fixed by #50
